tfx/proto/range_config.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package tfx.configs;
+
+// Describes a static window within the specified span numbers
+// [start_span_number, end_span_number]. If not all spans in the
+// range are available, processing may continue on a partial range as long as 
+// RangeConfig.min_spans are availiable.
+// Note that both numbers should be specified for StaticRange.
+message StaticRange {
+  int32 start_span_number = 1;
+  int32 end_span_number = 2;
```

Comment:
note that currently we only support start_span_number == end_span_number as examplegen only process one span at a time

Reply:
gotcha, i will update the behavior